packages/now-cli/src/commands/deploy/index.js

```diff
@@ -113,88 +85,5 @@ export default async ctx => {
     }
   }
 
-  if (localConfig) {
-    const { version } = localConfig;
-    const file = highlight(localConfig[fileNameSymbol]);
-    const prop = code('version');
-
-    if (version) {
-      if (typeof version === 'number') {
-        if (version !== 1 && version !== 2) {
-          const first = code(1);
-          const second = code(2);
-
-          output.error(
```

Comment:
We might want to keep validation to make sure `version` is either `undefined` or `2`. Because in the future when we add version=3, then we would want the old CLI to fail fast and not attempt to read the rest of the config. Maybe that validation needs to happen when reading the `vercel.json` file instead of deploying.